libsolidity/analysis/GlobalContext.cpp

```diff
@@ -123,7 +123,7 @@ vector<Declaration const*> GlobalContext::declarations() const
 {
 	vector<Declaration const*> declarations;
 	declarations.reserve(m_magicVariables.size());
-	for (ASTPointer<Declaration const> const& variable: m_magicVariables)
```

Comment:
Hmm.. strange that this was also changed by `modernize-use-override`. probably we need to remove this change.

Reply:
Yeah... and it's not a good change... this means that we'll really have to carefully review all of these :-).